indra-common/pom.xml

```diff
@@ -12,7 +12,7 @@
     <parent>
         <groupId>org.lambda3</groupId>
         <artifactId>indra-parent</artifactId>
-        <version>1.0.0-SNAPSHOT</version>
```

Comment:
We did not release version 1.0.0 hence we don't need to increment patch right now.